locales/th.schema.json

```
@@ -382,6 +382,17 @@
             "text": {
               "label": "ข้อความ"
             },
+            "text_alignment": {
+              "label": "การจัดวางเนื้อหาบนเดสก์ท็อป",
+              "option_1": {
+                "label": "ซ้าย"
+              },
+              "option_2": {
+                "label": "กึ่งกลาง"
+              },
+              "option_3": {
+                "label": "ขวา"
+            },
```

```suggestion
              "option_3": {
                "label": "ขวา"
              }
            },
```

To fix the JSON error we need to close `option_3`.